Gifski/MainWindowController.swift

```diff
@@ -159,7 +159,9 @@ final class MainWindowController: NSWindowController {
 			return
 		}
 
-		showInFinderButton.onAction = { _ in
+		conversionCompletedView.fileUrl = outputUrl
+
+		conversionCompletedView.showInFinderButton.onAction = { _ in
```

Comment:
You don't need to set the closure here as `conversionCompletedView` already get's the outputUrl, so you can just handle it inside `conversionCompletedView`.